media-tv/kodi/kodi-17.3-r2.ebuild

```diff
@@ -0,0 +1,286 @@
+# Copyright 1999-2017 Gentoo Foundation
```

Comment:
I figured that -r1 would be keyworded stable (since it only has the security fix) and -r2 would be ~unstable - hence the two revisions.